packages/yoastseo/src/parse/build/private/helpers/parseClassAttribute.js

```diff
@@ -0,0 +1,9 @@
+
+/**
+ * When splits a string of html classes into a set html classes.
+ * @param {string} classString The value of the class attribute.
+ * @returns {Set} A set containing all individual values of the class.
+ */
+export default function parseClassAttribute( classString ) {
+	return new Set( classString.split( " " ) );
```

Comment:
Question for CR: in hindsight, it is a little bit overdone to create a separate helper for a one-liner. Do you think this should be changed?